### PR TITLE
Import more test suites from Spark

### PR DESCRIPTION
**Pull request by rui-mo**
## What changes were proposed in this pull request?

Imported more test suites which we think are useful for Gluten from Spark.

## How was this patch tested?

Test on CI.

### CODE REVIEW

File: gluten-ut/src/test/scala/org/apache/spark/sql/GlutenSQLQueryTestSuite.scala [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql
+
+class GlutenSQLQueryTestSuite extends SQLQueryTestSuite with GlutenSQLTestsTrait {
```
Comment (rui-mo) [inline gluten-ut/src/test/scala/org/apache/spark/sql/GlutenSQLQueryTestSuite.scala:20]: There is a large scale of queries under https://github.com/apache/spark/tree/master/sql/core/src/test/resources/sql-tests/inputs for this test suite.

**Comment by rui-mo** [comment]
Will import `GlutenSQLQueryTestSuite` in a following PR due to maven abortion.

**Review by zhouyuan — APPROVED** [review]
👍

**Comment by rui-mo** [comment]
@zhouyuan Thank you for the review!

**Pull request merged**